pkg/util/iptables/save_restore.go

```diff
@@ -72,6 +73,20 @@ func GetChainLines(table Table, save []byte) map[Chain][]byte {
 	return chainsMap
 }
 
+// LoadIptablesSaveDataToBuffer load iptables-save output to buffer.
+// buffer can used exist, if no, will allocate  one
+func LoadIPtablesSaveDatasToBuffer(iptables Interface, buffer *bytes.Buffer, table Table) []byte {
```

Comment:
Nit: Make the buffer the last argument, since it is optional.

Reply:
done